packages/components/icons/index.tsx

```diff
@@ -5,3 +5,17 @@ export * from './Logo';
 export * from './PersonalizedLearningIcon';
```

Comment:
To pensando em remover esse arquivo. Ele ta dificultando nossa vida e ficando grande desnecessariamente. O que vcs acham sobre remover ele e importar os icones diretamente?

Outra opção é criar um componente de icones que mapeia uma props para esses icones, ai não precisamos importar coisas diferentes pra usar icones diferentes.

De toda forma, não estaria no escopo dessa tarefa, teria que ser feito em outra. Me conta ai o que vocês acham sobre.

Reply:
sera q da pra importar tipo o chakra com @?

Reply:
Como assim? não entendi bem sua pergunta, o @ é parte do nome da lib saca.